mslib/mscolab/models.py

```diff
@@ -42,7 +57,7 @@ class User(db.Model):
     password = db.Column(db.String(255), unique=True)
     registered_on = db.Column(db.DateTime, nullable=False)
     confirmed = db.Column(db.Boolean, nullable=False, default=False)
-    confirmed_on = db.Column(db.DateTime, nullable=True)
+    confirmed_on = db.Column(Awaredatetime, nullable=True, default=None)
```

Comment:
```suggestion
    confirmed_on = db.Column(AwareDateTime, nullable=True)
```

I think we should just leave out the default entirely, as before.